Dockerfile.sdk

```diff
@@ -105,6 +109,8 @@ WORKDIR /workspace/build
 RUN cmake -DCMAKE_INSTALL_PREFIX=/workspace/install \
           -DTRITON_VERSION=`cat /workspace/TRITON_VERSION` \
           -DTRITON_COMMON_REPO_TAG=${TRITON_COMMON_REPO_TAG} \
+          -DTRITON_CORE_REPO_TAG=${TRITON_CORE_REPO_TAG} \
```

Comment:
This change is also needed by the jetson_build script on Gitlab
          -DTRITON_CORE_REPO_TAG=${TRITON_CORE_REPO_TAG} \
          -DTRITON_BACKEND_REPO_TAG=${TRITON_BACKEND_REPO_TAG} \